### ISSUE TITLE
C++ header table layout

### ISSUE THREAD

**Issue by tomsmeding** [Thibaut/devdocs#688]
First of all: great work, this might replace the slow cppreference.com for me!

The table with overloads at the top of cppreference pages is notoriously hard to parse; I haven't looked at the source yet myself, but at least devdocs seems to get this wrong as well: compare https://devdocs.io/cpp/iterator/end and http://en.cppreference.com/w/cpp/iterator/end, for example, or compare https://devdocs.io/cpp/algorithm/min and http://en.cppreference.com/w/cpp/algorithm/min. Getting that table right would be a huge improvement! Like I said, I don't know how hard this is, but I just wanted to post an issue about it, since there wasn't one yet.

**Comment by Thibaut** [comment]
Thanks for the bug report. I'll look into this in the next C/C++ update. 

There's already lots of code to deal with similarly weird tables on cppreference, but I didn't know about those ones 😛
